src/form/FormInput.js

```diff
@@ -14,13 +14,22 @@ import normalize from '../helpers/normalizeText';
 const { width } = Dimensions.get('window');
 
 class FormInput extends Component {
+  constructor(props) {
+    super(props);
+    
+    this.getRef = this.getRef.bind(this);
+    this.focus = this.focus.bind(this);
+    this.blur = this.blur.bind(this);
+  }
+  
+  getRef() {
```

Comment:
use `getRef = () => {}` so you won't have to bind and you can also remove the contructor